data/kestrel.txt

```diff
@@ -188,6 +189,7 @@ mission "Kestrel Available"
 		has "kestrel available"
 	destination Wayfarer
 	on offer
+		log `Got a message from Charles Atinoda at Tarazed. The "Kestrel" I helped them design is available to purchase.`
```

Comment:
`The "Kestrel" I helped them design is available to purchase.` -> `The Kestrel is now available for purchase`